packages/js/src/httpClient.ts

```diff
@@ -44,9 +44,10 @@ export interface ClientOptions {
   orgId?: string;
 }
 
+// FIXME: this breaks on edge functions
 // The browsers don't have process.env, fake it
-const process: { env: Record<string, string | undefined> } =
-  typeof window === 'undefined' ? global.process : { env: {} };
+// const process: { env: Record<string, string | undefined> } =
+//   typeof window === 'undefined' ? global?.process : { env: {} };
```

Comment:
Commenting this out breaks browsers. Need to experiment with it more

Reply:
I pushed a commit which tries to avoid process.env on browsers, what do you think?